<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="PictureBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Изображение</string>
      </property>
      <widget class="QPushButton" name="prev_picture_button">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>0</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="next_picture_button">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>0</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>531</width>
         <height>331</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>529</width>
          <height>329</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="PictureBoxInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Информация</string>
        </property>
        <widget class="QLabel" name="token_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>171</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="analys_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Обработать</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>PictureBox</zorder>
   <zorder>PictureBoxInfo</zorder>
   <zorder>analys_button</zorder>
   <zorder>horizontalSpacer</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPicture_Analysis">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_picture_action"/>
    <addaction name="open_directory_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <addaction name="menuPicture_Analysis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_picture_action">
   <property name="text">
    <string>Выбрать изображение</string>
   </property>
  </action>
  <action name="open_directory_action">
   <property name="text">
    <string>Выбрать каталог</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
